schema205/cpp_translate.py

```diff
@@ -51,7 +51,7 @@ def translate_all_to_source(input_dir_path, output_header_dir, output_src_dir, c
         if 'RS' in file_name_root:
             factory_header = generate_factory_headers(file_name_root, 'RSInstance', container)
             dump(factory_header, os.path.join(output_header_dir, snake_style(file_name_root) + '_factory.h'))
-            factory_src = generate_factory_source(file_name_root, 'RSInstance', container)
+            factory_src = generate_factory_source(file_name_root, 'RSInstance', container, 'Ashrae205')
```

Comment:
`Ashrae205` is weird casing. Is this intentional?

Reply:
Dang, it should be `ASHRAE205` for the class name, shouldn't it? Just a brain blip.